embedded/object/schema.go

```diff
@@ -0,0 +1,23 @@
+package object
+
+import (
+	"github.com/codenotary/immudb/embedded/sql"
+)
+
+var ObjectPrefix = []byte{3}
```

Comment:
the sql engine already includes support for prefix isolation

document engine could just instantiate the engine with a different prefix. Similarly to what is done here but there should not be a need for fine-grained prefixes for tables, columns or indexes.

object prefix may be set in the database package together with the kv, sorted set and sql prefixes already declared.

I'd still create an object engine which can be a wrapper of the sql engine, so at the database package an object engine is instantiated without the knowledge of its internal implementation

Reply:
I see your point, I was in two minds about the same about using the prefixes or they have to be different at granular level. Thanks for the clarification, I'll just remove the granularities.

I agree, it will be better to add functionality to it separately